english/dream/fly/asia/asia.md

```diff
@@ -1,5 +1,67 @@
 You always wanted to visit some exotic countries,
-so you head of to asia.
```

Comment:
Casey isn't willing to merge this because you changed "of" to "off". I can see that you just fixed a typo, but she's not smart enough to know which changes are good and which are bad. I'll be happy to merge this if you remove all the extra blank lines.